### DISCUSSION TITLE
Files with accent marks "missing"

### DISCUSSION THREAD

**Discussion by phirestalker** [beetbox/beets#4508] [Q&A] [answered]
files with accent marks are not found by beets.
I think I may have caused this by changing my locale settings for another program. How can I get beets to recode the paths or rematch the existing files?

**Comment by sampsyo** [comment, marked as answer]
Try `beet update -p`! The `update` command removes database entries for deleted files (among other things), and its `--pretend` mode should fake it and just show you what it *would* do.

**Reply by phirestalker** [reply]
Oh crud, it seems I have a bunch of other things that need to be updated. I have to find a way to filter the results. The easiest way is to dump it to a text file and find/replace it to death.

Thanks for the tip.

**Reply by sampsyo** [reply]
It's possible that `beet update -p | grep deleted` could do the filtering you need!

**Reply by phirestalker** [reply]
The output looks like deleted is on a new line under the track info.

**Reply by JOJ0** [reply]
beet update -p | grep deleted -B 1

gives you the one line before deleted.

HTH

**Reply by phirestalker** [reply]
Yea, I thought there was a command switch like that, but didn't get around to looking it up. Thanks a bunch.

**Reply by phirestalker** [reply]
Some of these missing files do exist. Also, all of them have names with accent characters in them. I have never had trouble with unicode before. What should I do to troubleshoot this?

Hmm. ls command says the file is missing, but I see it over samba and can play it.

**Reply by phirestalker** [reply]
OK, when I use beet ls -p to feed the file location to ls it says no such file, but when I navigate there manually it is able to access the file.
So somehow beets is not using the right accent character even though they look the same?
I tried to `beet import -L` the album in question, but it quickly finished and didn't seem to do anything.

**Reply by phirestalker** [reply]
I just edited my question because I think I caused this issue by changing the system locale settings between two different UTF options.

**Reply by JOJ0** [reply]
I suggest you change it back to the original and leave an additional note like "update: ..." or similar because now all the answers so far are out of context and that's pretty misleading for others trying to help.
